deeprankcore/preprocess.py

```diff
@@ -34,7 +38,9 @@ def preprocess(
     queries: List[Query],
     prefix: Optional[str] = None,
     process_count: Optional[int] = None,
-    feature_modules: Union[List[ModuleType], str] = "all"):
+    combine_files: bool = True,
+    feature_modules: Union[List[ModuleType], str] = "all"
```

Comment:
why not use `None` as default make that use all features? Would look slightly cleaner that way, no? Or is it confusing that None is all?

Reply:
No I think it's fine, if it's None then all features are used. Updated now. We should then do the same with other parameters (e.g., node_feature and edge_feature in DataSet: when None, all are used) @DaniBodor

Reply:
In those cases, I think it's theoretically possible to use no node or edge features and actually set it to None (e.g. if you only want to use node features but no edge features).
On second thought, maybe here that would also be true. Not sure if there would be any situation where it would make sense to run a Query with absolutely no features at all, though.

Reply:
Let's keep None here, and the rest as it is. As soon as everything is documented, also different things are fine. I don't think there are any situations in which it would make sense to run a Query with no features at all, so None here makes sense to me.